content/ui/popup.html

```diff
@@ -1,49 +1,73 @@
-        <input type="password" id="cpassword" placeholder="Confirm Password">
-        <button type="submit" id="loginBtn">Set Password</button>
-      </form>
-      <div class="status"></div>
-    </div>
-    <div id="signInContainer">
-      <h1>Ping Kids Mode</h1>
-      <form id="loginForm">
-        <input type="password" id="loginPassword" placeholder="Password">
-        <button type="submit" id="loginBtn">Login</button>
-      </form>
-      <div class="status"></div>
-      <div style="display: flex; flex-direction: column; margin-bottom: 10px;">
-        <div class="checkbox-container">
-          <input type="checkbox" id="blockGamesCheckbox" style="margin: 5px;">
-          <label class="checkbox-label" for="blockGamesCheckbox">Block Games</label>
+  <div id="container" style="display: flex; flex-direction: column;">
+    <div id="popupContainers">
+      <div id="signUpContainer">
+        <div style="display: flex; justify-content: center;">
+          <h1 style="font-size: 18px; margin-bottom: 15px;">Kids mode</h1>
+        </div>
+        <form id="registerForm">
+          <input type="password" id="password" placeholder="Password">
+          <input style="margin-top: 10px;" type="password" id="cpassword" placeholder="Confirm password">
+          <button type="submit" id="loginBtn">Set password</button>
+        </form>
       </div>
-      <div class="checkbox-container">
-          <input type="checkbox" id="blockSocialMediaCheckbox" style="margin: 5px;">
-          <label class="checkbox-label" for="blockSocialMediaCheckbox" style="font-weight: bold; font-size: large;">Block Social Media</label>
+      <div id="signInContainer">
+        <div style="display: flex; justify-content: center;">
+          <h1 style="font-size: 18px; margin-bottom: 15px;">Kids mode</h1>
+        </div>
+        <form id="loginForm">
+          <input type="password" id="loginPassword" placeholder="Password">
+          <div style="display: flex; justify-content: start;">
+            <span id="resetPasswordLink" style="color: #a35e15; cursor: pointer; margin-left: 3px;">Reset
+              password</span>
+          </div>
+          <div style="display: flex; flex-direction: column; margin-bottom: 10px; margin-top: 15px;">
+            <div class="checkbox-container">
+              <input type="checkbox" id="blockGamesCheckbox" style="margin: 5px;">
+              <label class="checkbox-label" for="blockGamesCheckbox">Block games</label>
+            </div>
+            <div class="checkbox-container">
+              <input type="checkbox" id="blockSocialMediaCheckbox" style="margin: 5px;">
+              <label class="checkbox-label" for="blockSocialMediaCheckbox">Block social media</label>
+            </div>
+            <div class="session-time-container">
+              <label for="sessionTime" class="timerLabel">Set timer</label>
+              <input id="sessionTime" type="number" value="1" min="1" max="5" name="sessionTime">
```

Comment:
Why is max set to 5 here?

Reply:
You had said it in the previous comment, I thought the max hrs for which the kids mode should be allowed should be set to 5 hrs. If not then pls specify the min and max hrs to be set